lstchain/scripts/lstchain_mc_r0_to_dl1.py

```diff
@@ -42,12 +42,20 @@
                     help='Path to a configuration file. If none is given, a standard configuration is applied',
                     default=None
                     )
+parser.add_argument('--logger-level', '-l', action='store',
+                    dest='log_level',
+                    help='',
+                    default="NOTSET"
```

Comment:
Why default is "NOTSET" and not i.e. "ERROR"? Also how this is supposed to work with l57? `setLevel` takes `logging.LOG_LEVEL` object, not a string as an argument. See for example https://gitlab.cern.ch/cta-unige/sim_runner/-/blob/master/sim_runner/core/runner.py#L54-82

Reply:
From the documentation : Changed in version 3.2: The level parameter now accepts a string representation of the level such as ‘INFO’ as an alternative to the integer constants such as INFO.

I default as "NOTSET" because it means that the level used will be the one used by the root logger (should be WARNING by default)

Reply:
ok, fine then